### ISSUE TITLE
Crash on `metanorma --help/-h`

### ISSUE THREAD

**Issue by CAMOBAP795** [metanorma/metanorma-cli#52]
```
metanorma --help
/Library/Ruby/Gems/2.3.0/gems/metanorma-0.3.11/lib/metanorma/compile.rb:32:in `read': No such file or directory @ rb_sysopen - --help (Errno::ENOENT)
	from /Library/Ruby/Gems/2.3.0/gems/metanorma-0.3.11/lib/metanorma/compile.rb:32:in `options_extract'
	from /Library/Ruby/Gems/2.3.0/gems/metanorma-0.3.11/lib/metanorma/compile.rb:12:in `compile'
	from /Library/Ruby/Gems/2.3.0/gems/metanorma-cli-1.1.7/lib/metanorma/cli/compiler.rb:24:in `compile_file'
	from /Library/Ruby/Gems/2.3.0/gems/metanorma-cli-1.1.7/lib/metanorma/cli/compiler.rb:12:in `compile'
	from /Library/Ruby/Gems/2.3.0/gems/metanorma-cli-1.1.7/lib/metanorma/cli/compiler.rb:16:in `compile'
	from /Library/Ruby/Gems/2.3.0/gems/metanorma-cli-1.1.7/lib/metanorma/cli/command.rb:33:in `compile'
	from /Library/Ruby/Gems/2.3.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	from /Library/Ruby/Gems/2.3.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	from /Library/Ruby/Gems/2.3.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
	from /Library/Ruby/Gems/2.3.0/gems/metanorma-cli-1.1.7/lib/metanorma/cli.rb:56:in `start'
	from /Library/Ruby/Gems/2.3.0/gems/metanorma-cli-1.1.7/exe/metanorma:14:in `<top (required)>'
	from /usr/local/bin/metanorma:22:in `load'
	from /usr/local/bin/metanorma:22:in `<main>'
```

cc @ronaldtse @opoudjis

**Comment by ronaldtse** [comment]
The new command is `metanorma help`. @CAMOBAP795 or @abunashir can you help fix this?

**Comment by ronaldtse** [comment]
In any case, the command should fail gracefully if there is no file supplied.

**Comment by CAMOBAP795** [comment]
@ronaldtse I can help if need, once I will finish some pending tasks

**Comment by abunashir** [comment]
Thanks for reporting this one, Alex. I overlooked this but it should be fixed in #55
